source/agora/consensus/pool/Enrollment.d

```diff
@@ -17,6 +17,7 @@
 module agora.consensus.pool.Enrollment;
 
 import agora.common.ManagedDatabase;
+import agora.common.Set;
```

Comment:
Obviously this is not used in this commit.

Reply:
Removed.